net.crossfireatlas.crossfire.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>net.crossfireatlas.crossfire</id>
+	<name>Crossfire RPG</name>
+	<summary>Roguelike multi-user RPG</summary>
+	<metadata_license>CC-BY-SA-3.0</metadata_license>
+	<project_license>GPL-2.0</project_license>
+	<!-- Description taken from https://crossfire.real-time.com -->
+	<description>
+                <p>An introduction to the game of Crossfire, and links to other sites and what they have to say about the game.</p>
+                <p>In the words of its original author Frank Tore Johansen:</p>
```

Comment:
Does it make sense to make this a quote?

Reply:
The quote kind of describes the game (as the original author put it). But I guess it can be stated without quoting anyone. So, removed the quote part.